test-project/Assets/EditmodeTests/GameObjectRepresentation/MonoBehaviourActivationManager/CommandHandlerBehaviourActivationTests.cs

```diff
@@ -0,0 +1,19 @@
+using Generated.Improbable.Gdk.Tests.ComponentsWithNoFields;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.GameObjectRepresentation.EditModeTests.MonoBehaviourActivationManagerTests
+{
+    [TestFixture]
+    public class CommandHandlerBehaviourActivationTests :
```

Comment:
Is this file just stubbed out?

Reply:
no, it runs the tests in the base class with this configuration

Reply:
I'll add a comment

Reply:
Ah I missed it derived from not the base class but the authority base class